File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if (s.size() != t.size())
            return false;
        
        std::sort(s.begin(), s.end());
        std::sort(t.begin(), t.end());

        for (std::size_t idx = 0; idx < s.size(); ++idx) {
            if (s[idx] != t[idx])
                return false;
        }

        return true;
    }
};

File: Data Structures & Algorithms/non-overlapping-intervals/submission-0.py
class Solution:
    def eraseOverlapIntervals(self, intervals: List[List[int]]) -> int:
        intervals.sort(key=lambda x: x[1])
        c_start, c_end = intervals[0]
        removed = 0

        for start, end in intervals[1:]:
            if start < c_end:
                removed += 1
            else:
                c_start = start
                c_end = end

        return removed